quickevent/app/plugins/CardReader/CardReader.pro.orig

```diff
@@ -1,50 +0,0 @@
-message(including plugin $$PWD)
```

Comment:
tohle vubec nepatri do gitu

Reply:
Ja jsem nalezene .orig soubory mazal. 
Proc je tam Fanda commitl to netusim.

Reply:
@arnost00  Fanda muze commitnout jenom to co je v pull requestu, takze ty soubory jsi tam musel dat ty

Reply:
jinak moc nerozumim smyslu tohoto commitu, co to vlastne meni na fungovani QE?

Reply:
uz vim, nejde to udajne slinkovat

Reply:
je treba vyresit konflikt

Reply:
ty orig soubory přidal do gitu @fvacek když o vánocích předělával qml na c++ (pravděpodobně si je tam chtěl ponechat jako zálohu, kdyby něco nefungovalo tak, jak má), @arnost00 je nyní chce odstranit

Reply:
Plugin Receipts je zavisly na CardReader. A Ota pridal opacnou zavislost (tj. ze by CardReader byl zavisly na Receipts). Coz samozrejme nejde ve Windows slinkovat. Predelal jsem to na invokeMethod, ktere se uz v CardReader vyskytovaly.

Reply:
> je treba vyresit konflikt

Tak to si nejdriv musim projit co vse tam mezitim kdo zmenil. V dobe kdy jsem to zadaval tam jeste zadny konflikt nebyl.